experimental/examples/tree-sample/inventory-app/src/schema.ts

```diff
@@ -4,43 +4,32 @@
  */
 
```

Comment:
Looks like this file was overlooked in the migration to the updated schema syntax.

(Fixed by copying the 'schema.ts' from the unit test.)